dev/src/index.ts

```diff
@@ -1372,15 +1372,18 @@ export class Firestore {
             request
           );
           const stream = gapicClient[methodName](request, callOptions);
-          const logStream = through2.obj(function(this, chunk, enc, callback) {
-            logger(
-              'Firestore.readStream',
-              requestTag,
-              'Received response: %j',
-              chunk
-            );
-            this.push(chunk);
-            callback();
+          const logStream = new PassThrough({
```

Comment:
This can be a constructed by a `responseLogger` function that takes `methodName` and `requestTag`.

Reply:
Apologies - I do not know this code base terribly well :) I was just trying to do a 1 for 1 replacement of `through2` with Transform streams, now that we've dropped node.js 6.  Could you spell out the line of code you think I should be using here?

Reply:
Oh, I was just suggesting that these two could be combined. We can do that in another pass.